docs/connectors/sources/iceberg.md

```diff
@@ -0,0 +1,314 @@
+
+#### S3 FileIO configuration
+
+| Property                    | Type   | Description   |
+|-----------------------------|--------|---------------|
+| `glue.warehouse`*           | string | Location for table metadata. Example: `s3://my-data-warehouse/tables/`|
+| `s3.access-key-id`          | string | S3 access key id.|
+| `s3.s3.secret-access-key`   | string | S3 secret access key.|
+| `s3.endpoint`               | string | Configure an alternative endpoint of the S3 service for the FileIO to access. This could be used to use S3 FileIO with any S3-compatible object storage service that has a different endpoint, or access a private S3 endpoint in a virtual private cloud.|
+| `s3.region`                 | string | S3 region.|
+| `s3.session-token`          | string | S3 session token. This is required when using temporary credentials.|
+| `s3.allow-anonymous`        | string | Set to `"true"` to skip signing requests (e.g., for public buckets).|
+| `s3.disable-ec2-metadata`   | string | Set to `"true"` to skip loading the credential from EC2 metadata (typically used in conjunction with `s3.allow-anonymous`).|
+
+#### GCS FileIO configuration
+
+| Property                    | Type   | Description   |
+|-----------------------------|--------|---------------|
+| `gcs.project-id`            | string | Google Cloud Project ID.|
+| `gcs.service.path`          | string | Google Cloud Storage endpoint.|
+| `gcs.no-auth`               | string | Set to `"true"` to allow unauthenticated requests.|
+| `gcs.credentials-json`      | string | Google Cloud Storage credentials JSON string, base64 encoded.|
+| `gcs.oauth2.token`          | string | String representation of the access token used for temporary access.|
+
+## Data type mapping
+
+The following table lists supported Iceberg data types and corresponding Feldera types.
+
+| Iceberg type                | Feldera SQL type | Comment       |
+|-----------------------------|------------------|---------------|
+| `boolean`                   | `BOOLEAN`        |               |
+| `int`                       | `INT`            |               |
+| `long`                      | `BIGINT`         |               |
+| `float`                     | `REAL`           |               |
+| `double`                    | `DOUBLE`         |               |
+| `decimal(P,S)`              | `DECIMAL(P, S)`  | The largest supported precision `P` is 28.|
+| `date`                      | `DATE`           |               |
+| `time`                      | `TIME`           |               |
+| `timestamp`                 | `TIMESTAMP`      |               |
+| `string`                    | `STRING`         |               |
+| `uuid`                      | `STRING`         |               |
+| `fixed(L)`                  | `BINARY(L)`      |               |
+| `binary`                    | `VARBINARY`      |               |
+
+Types that are currently not supported include nested data types (`struct`s,
```

Comment:
are these iceberg types or feldera types?

Reply:
both, but I clarified that they are Iceberg types.